dev/Telemetry/MicrosoftTelemetry.h

```diff
@@ -1,549 +1,86 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.
+﻿/* ++
 
-/* ++
+Copyright (c) Microsoft Corporation.  All rights reserved.
```

Comment:
I think the license header should remain the same as before.

Reply:
Hmm, good question. I'm using the official stubbed header from the Windows repo now, I'll inquire with them if it should be changed to include the license.

Reply:
Updated